docs/mixed_precision.md

```diff
@@ -0,0 +1,22 @@
+# 混合精度训练
+
+PLSC支持混合精度训练。使用混合精度训练可以提升训练的速度，同时减少训练使用的内存。
+
+可以通过下面的代码设置开启混合精度训练：
+
+```python
+from __future__ import print_function
+import plsc.entry as entry
+
+def main():
+    ins = entry.Entry()
+    ins.set_mixed_precision(True, 1.0)
+    ins.train()
+
+if __name__ == "__main__":
+    main()
+```
+其中，`set_mixed_precision`函数为设置混合精度训练是否开启。
```

Comment:
Give the definition of set_mixed_precision to clearly show its parameters.

Reply:
done. thanks